### PR TITLE
Simulated Test Updates

### PR DESCRIPTION
**Pull request by PhilD71**
### Description
Blocks for primitives + removes start ball placement command which was causing GC to send Halt command

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**
Will update once CI passes
- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/simulated_tests/simulated_test_fixture.py [hunk 1]
```diff
@@ -167,6 +170,10 @@ def __runner():
                             RobotStatus, robot_status
                         )
 
+                        _ = self.primitive_set_buffer.get(
```
Comment (jonathanlew) [inline src/software/simulated_tests/simulated_test_fixture.py]: why don't we do anything with the primitiveset we get?

Comment (PhilD71) [reply src/software/simulated_tests/simulated_test_fixture.py]: The idea was to block until we get it I think

Comment (jonathanlew) [reply src/software/simulated_tests/simulated_test_fixture.py]: please figure out what this is about and add a comment about it

**Review by jonathanlew — DISMISSED** [review]
LGTM!

**Review by itsarune — COMMENTED** [review]
lgtm pending minor grammar nit

File: src/software/simulated_tests/simulated_test_fixture.py [hunk 2]
```diff
@@ -167,6 +170,11 @@ def __runner():
                             RobotStatus, robot_status
                         )
 
+                        # blocking wait for primktivies to keep world in sync with them
```
Comment (itsarune) [inline src/software/simulated_tests/simulated_test_fixture.py:173]: I think the grammar nit is still herr

**Review by itsarune — COMMENTED** [review]
just the minor nit

File: src/software/simulated_tests/simulated_test_fixture.py (hunk 2 above)
Comment (PhilD71) [reply src/software/simulated_tests/simulated_test_fixture.py:173]: Okay something weird is going on

File: src/software/simulated_tests/simulated_test_fixture.py [hunk 3]
```diff
@@ -167,6 +170,11 @@ def __runner():
...
+                        self.primitive_set_buffer.get(
+                            block=True, timeout=WORLD_BUFFER_TIMEOUT
+                        )
```
Comment (jonathanlew) [inline src/software/simulated_tests/simulated_test_fixture.py:176]: it's weird that this blocking wait is happening in the exception, doesn't seem like it'd affect the normal code path

**Review by jonathanlew — COMMENTED** [review]
just thought of something ^

**Pull request closed without merging**